contrib/blueflood-docker-compose/docker-compose-cass-es.yml

```diff
@@ -0,0 +1,61 @@
+# using Docker-Compose Version 3
+version: '3'
+
+services:
+  # Configuration for seed cassandra node. Also, this is the only node
+  cassandra:
+    container_name: cassandra-node
+    image: cassandra:2.1
+    ports:
+        - "7000:7000"
+        - "7001:7001"
+        - "9042:9042"
+        - "9160:9160"
+        - "7199:7199"
+
+    # Exposing ports for inter cluster communication.
+    expose:
+      - 7000 # Intra-node communication
+      - 7001 # TLS intra-node communication
+      - 7199 # JMX
+      - 9042 # CQL
+      - 9160 # Thrift service
+
+    command: bash -c 'if [ -z "$$(ls -A /var/lib/cassandra/)" ] ; then sleep 0; fi && /docker-entrypoint.sh cassandra -f'
+
+    # Network for the nodes to communicate
+    networks:
+      - dc1ring
+
+    # Maps cassandra data to a local folder. This preserves data across
+    # container restarts. Note a folder 'cassdata' gets created locally in HOME directory.
+    volumes:
+      - ${HOME}/cassdata:/var/lib/cassandra
+
+    # Docker container environment variable. We are using the
+    # CASSANDRA_CLUSTER_NAME to name the cluster. This needs to be the same
+    # across clusters. We are also declaring that 'cassandra' is a seed node.
+    environment:
+      - CASSANDRA_CLUSTER_NAME=dev_cluster
+          - CASSANDRA_SEEDS=cassandra
```

Comment:
The indent on this line seems to have an extra few spaces. I'm surprised yaml/compose didn't complain.

Reply:
Just fixed it. Good catch. I am surprised too. Unless this indentation somehow by mistake got added after the run.